test/retransmission_test.exs

```diff
@@ -0,0 +1,71 @@
+defmodule ExDTLS.RetransmissionTest do
+  use ExUnit.Case, async: true
+
+  test "check retransmission" do
+    {:ok, rx_pid} = ExDTLS.start_link(client_mode: false, dtls_srtp: true)
+    {:ok, tx_pid} = ExDTLS.start_link(client_mode: true, dtls_srtp: true)
+    {:ok, packets} = ExDTLS.do_handshake(tx_pid)
+    assert :ok == loop({rx_pid, false}, {tx_pid, false}, packets)
+  end
+
+  @tag timeout: :infinity
+  @tag :capture_log
+  @tag :long_running
+  test "check reaching timeout limit" do
+    {:ok, rx_pid} = ExDTLS.start_link(client_mode: false, dtls_srtp: true)
+    {:ok, tx_pid} = ExDTLS.start_link(client_mode: true, dtls_srtp: true)
+    {:ok, packets} = ExDTLS.do_handshake(tx_pid)
+
+    Process.flag(:trap_exit, true)
+
+    try do
+      process_until_raise(rx_pid, packets, 1)
+    catch
+      :exit, _msg ->
+        assert true
+    end
+  end
```

Comment:
```suggestion
  @tag :long_running
  test "check reaching timeout limit" do
    Process.flag(:trap_exit, true)

    {:ok, rx_pid} = ExDTLS.start_link(client_mode: false, dtls_srtp: true)
    {:ok, tx_pid} = ExDTLS.start_link(client_mode: true, dtls_srtp: true)
    {:ok, packets} = ExDTLS.do_handshake(tx_pid)
    {:handshake_packets, _packets} = ExDTLS.process(rx_pid, packets)

    # ignore retransmissions
    Process.sleep(ExDTLS.get_max_retransmit_timeout() * 2)

    assert_receive({:EXIT, ^tx_pid, {%RuntimeError{message: _msg}}, _stack}, 2000)
    assert_receive({:EXIT, ^rx_pid, {%RuntimeError{message: _msg}, _stack}}, 2000)
  end
```

Let's add also a function `get_retransmission_max_timeout`
```elixir
  @doc"""
  Returns max retransmission timeout after which `ExDTLS` will raise an error.
  
  Timer starts at one second and is doubled each time `ExDTLS` does not receive a response.
  After reaching `@max_retransmission_timeout` `ExDTLS` will raise an error.
  """
  @spec get_max_retransmit_timeout() :: non_neg_integer()
  def get_max_retransmit_timeout(), do: @max_retransmit_timeout
```

Reply:
Why do we need this function? Why not add only doc?